docs/spec/abci/implementations.md

```diff
@@ -0,0 +1,35 @@
+# ABCI Implementations
```

Comment:
This is great but seems its still largely a duplicate of https://github.com/tendermint/tendermint/blob/develop/docs/spec/abci/client-server.md

Can we just make sure that file contains anything from here it might be missing?

Reply:
fixed